[DOCS-5726] API Catalog updates (#20039)
DataDog/documentation: content/en/tracing/api_catalog/monitor_apis.md

The latest monitor status is displayed in the Explorer, and you can see more information about why a monitor is alerting, how to address it, whether launch an investigation by clicking through to view the monitors.

{{< img src="tracing/api_catalog/api-catalog-monitor.png" alt="The monitor status menu and Create Monitor button in API Catalog Explorer" style="width:40%;" >}}

You can also create a monitor on Latency or Error Rate directly from API Catalog by clicking **Add Monitor**. Read [Alerting][1] for additional information about setting up and managing monitor alerts.

## Track and improve API test coverage

Using Synthetic API tests, you can set up automated scheduled tests of your endpoints that alert you if the tests fail and letting you diagnose and fix the problem. 

The Explorer page shows you which of your endpoints have, or more importantly _don't_ yet have tests created for them, and whether any of them are failing. If tests are failing, click through to investigate why.

{{< img src="tracing/api_catalog/api-catalog-tests.png" alt="The API tests status menu and Create Test button in API Catalog Explorer" style="width:40%;" >}}

> Click **+ API Test** to create another synthetic API test for this endpoint. Read the documentation about [Synthetic HTTP API Tests][3] for more information about setting up tests.

Reviewer: ```suggestion
Click **+ API Test** in the **API TESTS column** to create another synthetic API test for this endpoint. Read the documentation about [Synthetic HTTP API Tests][3] for more information about setting up tests.
```

## Find and close security gaps

> Powered by Datadog [Application Security Management (ASM)][4], the Security Signals column in the Explorer shows you if ASM has detected threats to the services associated with your endpoints. Sort the table by the column to quickly see which of your endpoints is most affected. Click **View in ASM** to investigate the and address the threats and vulnerabilities in your code or your upstream dependencies.

Reviewer: ```suggestion
Powered by Datadog [Application Security Management (ASM)][4], the **SECURITY SIGNALS** column in the Explorer shows you if ASM has detected threats to the services associated with your endpoints. Sort the table by this column to see which of your endpoints are most affected. Click **View in ASM** to investigate and address the threats and vulnerabilities in your code or your upstream dependencies.
```
